examples/flutter_firebase_login/lib/sign_up/cubit/sign_up_cubit.dart

```diff
@@ -2,6 +2,7 @@ import 'package:authentication_repository/authentication_repository.dart';
 import 'package:bloc/bloc.dart';
 import 'package:equatable/equatable.dart';
 import 'package:flutter_firebase_login/authentication/authentication.dart';
+import 'package:flutter_firebase_login/authentication/models/confirmedPassword.dart';
```

Comment:
this import should not be needed because we should export `confirmed_password.dart` from the `authentication.dart` barrel file.